src/main/java/school/lemon/changerequest/java/Employees/Accountant.java

```diff
@@ -0,0 +1,29 @@
+package school.lemon.changerequest.java.Employees;
+
+/**
+ * Created by lbrdev on 04.01.2017.
+ * Project: oop.pr1
+ */
+public class Accountant extends Employee {
+    private static int countAccountant;
+
+    public Accountant(String name) {
+        super(name);
+    }
+
+    @Override
+    public double calculateSalary() {
+        if (getRatioOfHours() >= 1.) {
+            return getSalary();
+        }
+        return getRatioOfHours() * getSalary();
+    }
+
+    public int overallSalary(Employee[] employees) {
```

Comment:
I'd prefer to store array/list of employees in the field and do not pass them through argument.